app/views/users/terms.html.erb

```diff
@@ -28,27 +30,36 @@
     <%= render :partial => "terms" %>
   </div>
 
+  <div class="deemphasize">
+    <p id="contributorGuidance">
+      <%= raw t '.guidance',
+                :summary => 'https://www.osmfoundation.org/wiki/License/Contributor_Terms_Summary',
+                :translations => 'https://www.osmfoundation.org/wiki/License/Contributor_Terms/Informal_Translations' %>
+    </p>
+  </div>
+
+  <label class="standard-label">
+    <%= t 'layouts.tou' %>
+  </label>
+  <p class="deemphasize"><%= t '.tou_explain' %></p>
   <div class="form-row">
-    <label for="user_consider_pd">
-      <%= check_box('user', 'consider_pd') %>
-      <%= t '.consider_pd' %>
+    <label for="read_tou">
+      <%= check_box_tag 'read_tou',1,false %>
```

Comment:
There's no need for the last two arguments - those are the default values.

Reply:
Fixed in https://github.com/openstreetmap/openstreetmap-website/pull/2028/commits/5dfbc92ef60112c55462bab75898c25d31d6bc84